### ISSUE TITLE
Add support for null values with PrepareQuery

### ISSUE THREAD

**Issue by DragonGER** [intercept/intercept-database#19]
Sometimes you need to set columns to null in sql. Imagine you have a killtracker that saves killer and victims uid but you also want to track AI kills. In this case i would set the field null in case of AI.

Let's look at an example.
YAML
```javascript
insertKill: INSERT INTO kill_log (killer_id, victim_id, weapon, distance) VALUES (?,?,?,?);
```
Let's have a look at the sqf code i would like to use:
```javascript
params [
	["_killerId", nil, ["", nil]],
	["_victimId", nil, ["", nil]],
	["_weapon", nil, ["", nil]],
	["_distance", 0, ["", 0]]
];
private _query = dbPrepareQueryConfig ["insertKill", [_killerId, _victimId, _weapon, _distance]];
database dbExecuteAsync _query;

true
```
So far dbPrepareQueryConfig only supports SKALAR, STRING and BOOL Values but no nil values.
It would be pretty awesome to add translate all null reference arma3 types (objNull, groupNull, nil, null, ...) to the sql null. I think this will be pretty handy in a lot of cases :)

**Comment by dedmen** [comment]
[idb_x64_30062019.zip](https://github.com/intercept/intercept-database/files/3343206/idb_x64_30062019.zip)
Build with this fixed

**Comment by dedmen** [comment]
Fix was included in 1.4
